src/In.ProjectEKA.OtpService/Otp/Logger/Log.cs

```diff
@@ -0,0 +1,44 @@
+
+namespace In.ProjectEKA.OtpService.Otp.Logger
+{
+    using System;
+    using Serilog;
+    
+    public static class Log
+    {
+        private static ILogger Logger
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'Logger'.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4954915)